src/angular/overlay/overlay-container.ts

```diff
@@ -0,0 +1,92 @@
+import { CdkPortalOutlet, type ComponentPortal, type TemplatePortal } from '@angular/cdk/portal';
+import {
+  type OnDestroy,
+  Component,
+  type ComponentRef,
+  effect,
+  type EmbeddedViewRef,
+  inject,
+  ViewChild,
+} from '@angular/core';
+import { outputToObservable } from '@angular/core/rxjs-interop';
+import {
+  SbbOverlayConfig,
+  SbbOverlayContainerBase,
+  SbbOverlayState,
+} from '@sbb-esta/lyne-angular/core/overlay';
+import { type Observable, Subject } from 'rxjs';
+
+import { SbbOverlay } from './overlay';
+
+@Component({
+  selector: 'sbb-overlay[sbb-overlay-container]',
```

Comment:
Curious, why is the attribute sbb-overlay-container needed? where is it set or when should it be used? I didn't find it in documentation